### ISSUE TITLE
Add custom styles Support

### ISSUE THREAD

**Issue by juanmako** [wyozi/react-pptx#673]
Thanks for such a great package! I have been working with it, and recently I tried to add some styles to the Image component, but I know it is not supported currently. 

Nevertheless, I think it is a good idea to support a wrapper tag, or even better, just an identifier, and add later a custom modification in the html document. 

So, I think it's a great idea to just support an id field to finally have a custom identifier in the actual HTML document.

<img width="381" alt="Screenshot 2023-08-12 at 8 45 11 AM" src="https://github.com/wyozi/react-pptx/assets/137253810/49bfa529-8b53-4b68-abe3-d6ac0a42f33e">

**Comment by wyozi** [comment]
Presumably this would only affect the HTML preview? Would be interesting to know what kind of styling do you need to apply to an image in general :)

**Comment by juanmako** [comment]
You are right! After doing it with a useEffect in React, that would only affect the preview 
Nevertheless, the HTML prop does not work on react-pptx components.

Thanks a lot for your answer!

**Comment by juanmako** [comment]
Thanks a lot for your answer!

**Comment by juanmako** [comment]
Thanks a lot for your answer!

**Comment by wyozi** [comment]
I'll keep this open until the `html` prop is implemented :)

**Comment by malavshah9** [comment]
@wyozi Didn't get fully on what to implement here. Could you elaborate in detail like what changes we wanted to make in API?
